packages/gatsby-theme-docs/src/utils/markdown-2-react.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import markdown2React from './markdown-2-react';
+
+describe('markdown2React', () => {
+  it('should return a react component from a markdown string', async () => {
+    const markdownString = '# This is a markdown title';
+    const expectedReactElement = React.createElement('h1', { key: 'h-1' }, [
+      'This is a markdown title',
+    ]);
+
+    expect(markdown2React(markdownString)).toEqual(expectedReactElement);
```

Comment:
I think it would be enough to just check that the return value is a react element:

```suggestion
    expect(React.isValidElement(markdown2React(markdownString))).toBeTruthy();
```

This avoids the need to test the implementation of the parsing library, which is not useful to us